### ISSUE TITLE
trello like drag and drop with multiple lists

### ISSUE THREAD

**Issue by domnulnopcea** [taye/interact.js#855]
hello,

is there an example how I could use this library to implement a trello like board?

is this library capable of something like this?

thanks a lot

**Comment by owenfar** [comment]
As far as I know interact.js doesn't support such a feature "out-of-the-box". This library is capable of doing that and much more, and there seems to be plans that might easily allow you to implement something similar to trello. For now, such an implementation requires further custom code from your side. 

If you're not very technical, I'de recommend you find a library that does exactly that (there are a few if you look around). On the other hand, if you're comfortable with JS, you can do something like trello and tweak it exactly to your likings.

**Comment by taye** [comment]
@owenfar that's right. Though I'm working on a set of paid plugins which includes list reordering. It's still a work in progress, but you can find out more at https://interactjs.io/pro.

**Comment by timothyallan** [comment]
> @owenfar that's right. I'm working on a set of paid plugins which includes list reordering. It's still a work in progress, but you can find out more at https://interactjs.io/pro.

I'd love to throw some money your way @taye, this project is great! I signed up for the newsletter. Are you at a years, months, or weeks ETA for any sort of Pro beta?

**Comment by taye** [comment]
@timothyallan I'm at months, *possibly* weeks. The major things to work on are service reliability (of private npm registry, cdn and account management), documentation :grimacing:, licensing + TOS :weary:, and a few bug fixes + automated tests.

**Comment by owenfar** [comment]
@taye That's really great to hear! I've been looking forward for some of the pro-features ever since you teased about them :) Wish you all the best with the progress. Thank you for your time to build something so great, we all appreciate your effort.

**Comment by tylermercer** [comment]
Also curious of the status of sorting/reordering.
